src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return (days * 40) - 20;
```

Comment:
numbers 40, 20 and 50 are not clear.
Create variables with definitive names.

Reply:
Noted. Have changed it kindly check.